public/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script src="%PUBLIC_URL%/env-config.js"></script>
+    <meta charset="utf-8" />
+    <link rel="icon" href="https://st2.depositphotos.com/4362315/7819/v/950/depositphotos_78194060-stock-illustration-medical-logo-health-care-center.jpg" />
```

Comment:
is this a purchased photo? If so then store them directly.

Reply:
Create a new favicon icon and added the reference here.